src/ui/components/Card/Card.tsx

```diff
@@ -50,14 +52,16 @@ export const CropCard = forwardRef<HTMLDivElement, CropCardProps>(
 
 export const WaterCard = forwardRef<HTMLDivElement, WaterCardProps>(
   function WaterCard(props, ref) {
-    return <CardCore {...props} ref={ref} />
+    const cardInteractionProps = useCardInteractions(props)
```

Comment:
As a matter of stylistic preference, insert a newline before the `return` statements in the modified functions that have statements in addition to the `return`. For instance, instead of this:

```ts
    const interactionProps = useCardInteractions(props)
    return (
      <CardCore {...props} {...interactionProps} ref={ref}>
```

Do this:

```ts
    const interactionProps = useCardInteractions(props)

    return (
      <CardCore {...props} {...interactionProps} ref={ref}>
```

This does not apply to functions that don't contain anything beyond a `return` statement. This is a project-wide stylistic preference. Keep this update scoped to only what this PR touches. However, be sure to capture this stylistic preference in the `AGENTS.md` file and include that in this PR.